apps/meteor/server/modules/notifications/notifications.module.ts

```diff
@@ -102,15 +102,12 @@ export class NotificationsModule {
 			}
 
 			const canAccess = await Authorization.canAccessRoom(room, { _id: this.userId || '' }, extraData);
```

Comment:
You can remove this call to canAccessRoom since it'll be called inside of the canReadRoom function. Just make sure to pass the extraData to it as well.